src/main/java/io/quarkus/backports/BackportsResource.java

```diff
@@ -62,6 +63,10 @@ public TemplateInstance backports(@NotNull(message = "Invalid Milestone")  @Rest
     public String markAsBackported(@NotNull(message = "Invalid Milestone") @RestPath Milestone milestone,
                                    @NotNull(message = "Invalid Pull Request") @RestPath PullRequest pullRequest) throws IOException {
         gitHub.markPullRequestAsBackported(pullRequest, milestone);
+        Log.info("    Backported PR to " + milestone.title + ": " + pullRequest.url);
```

Comment:
```suggestion
Log.info("    Backported PR to " + milestone.title + ": " + pullRequest.url);
```
I think Guillaume wanted to indent just details about commits

Reply:
Ah ok, good point.